packages/eslint-plugin-clarity-migration/src/rules/no-clr-button/index.ts

```diff
@@ -0,0 +1,61 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils';
+import { parse } from 'node-html-parser';
+import { getDecoratorPropertyValue } from '../utils';
+import { HTMLElement } from 'eslint-html-parser';
+
+export const createESLintRule = ESLintUtils.RuleCreator(() => ``);
+
+export type MessageIds = 'clrButtonFailure';
+
+export default createESLintRule({
+  name: 'no-clr-button',
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Disallow use of clr-button',
+      category: 'Best Practices',
+      recommended: 'warn',
```

Comment:
Any thoughts as to where we might put url's to written guides? E.g - github wiki or website release notes?

Reply:
I think these should be part of the error messages.
```js
messages: {
	clrButtonFailure: 'Using clr-button is not allowed! See _link_.'
}
```

Reply:
We don't have links yet but I agree this is how we'd do it generally speaking.